lms/static/js/certificates/views/certificate_whitelist.js

```diff
@@ -32,7 +32,12 @@
                 render: function(){
                     var template = this.loadTemplate('certificate-white-list');
                     this.$el.html(template({certificates: this.collection.models}));
-
+                    if(this.collection.isEmpty()){
+                        this.$("#generate-exception-certificates").addClass("is-disabled").attr('disabled', 'disabled');
```

Comment:
is it required to add the `disabled` attribute if we are applying the class `is-disabled` ?